setup.py

```diff
@@ -41,53 +41,33 @@
         long_description = f.read()
 
     install_requirements = [
+        "vivarium_dependencies[numpy,pandas,pyyaml,scipy,click,tables,loguru,pyarrow,networkx]",
```

Comment:
Big question...should I be pinning vivarium_dependencies just like we pinned `vivarium_build_utils` below? It seems like we should not pin by default and then only if needed in the future?

Reply:
No, I don't think we want to pin.